### PR TITLE
(Update) Twitter: Demo and Documentation

### LINKED ISSUE #7727: Twitter no longer requires the Widget-ID
Hi,
Twitter no longer requires or allows the creation of a  "widget ID".
The Drupal Wetkit Variant depends on the javascript code.

Can this be updated to not require the Widget ID?

Thanks
Grant

More info can be found at twittercommunity here:
https://twittercommunity.com/t/how-can-i-get-a-twitter-widget-id/68917

and at

Wetkit downstream @ the Drupal Variant here:
https://www.drupal.org/node/2778507

### PR DESCRIPTION
**Pull request by ghost**
Fixes #7727

### CODE REVIEW

**Comment by kodecount** [comment]
Also only the the content I'm changing/updating requires french.

**Comment by kodecount** [comment]
Added the French pages which well need to be translated.

**Comment by lucas-hay** [comment]
Discussed in PR review meeting., Once French pages (with english text on them) are added, we will merge and translate in the future. (Do not forget your span and lang tags)

**Comment by EricDunsworth** [comment]
FYI here's an example of a demo that's already doing it:
http://wet-boew.github.io/wet-boew/docs/ref/footnotes/footnotes-fr.html

Btw from what I've seen, the children of ``<div lang="en">`` aren't usually indented in that kind of scenario.

**Comment by kodecount** [comment]
Thanks. I'll try to get too this today. Been super busy. Getting ready for a possible move.

**Comment by kodecount** [comment]
All updated.

File: src/plugins/twitter/twitter-fr.hbs [hunk 1]
```diff
@@ -12,13 +12,36 @@
 ---
 <span class="wb-prettify all-pre hide"></span>
 
+<div lang="en">
```
Comment (duboisp) [inline src/plugins/twitter/twitter-fr.hbs]: Can you move the this line after the french section bellow after line 21

**Review by duboisp — CHANGES_REQUESTED** [review]

**Review by EricDunsworth — CHANGES_REQUESTED** [review]
I noticed a few issues related to extra tab characters within ``<pre>`` elements and a couple of ``<a>`` elements spread out across several lines.

**Comment by lucas-hay** [comment]
@kodecount will you be able to clean up the requests above? Very minor changes.

**Comment by kodecount** [comment]
Sorry I was without full internet for awhile. I'm back online so I'll try to do it by tomorrow night.

**Comment by kodecount** [comment]
I'm not able to fix this sorry laptop is not working. Well try to get laptop working soon, but as for now I'm unsure when that we'll be.

**Comment by lucas-hay** [comment]
@kodecount no worries Kody, we will wait until you have time. Thanks again.

**Comment by kodecount** [comment]
It should be updated.

**Review by duboisp — APPROVED** [review]

**Pull request merged**